main.go

```diff
@@ -156,6 +156,8 @@ func main() {
 					log.Fatal("Failed to migrate store schema: ", err)
 				}
 
+				return err
```

Comment:
This err will always be `nil`. This is the one that `store.NewMySQLStore returns`, and it's checked just afterwards. Instead return here `nil` and change all the `log.Fatal` to returns the errors, like here:

```go
if err != nil {
    return fmt.Errorf("failed to create Store: %v", err)
}
```